src/Factory/OrmFactory.php

```diff
@@ -65,9 +68,9 @@ public function __invoke(
                 if (!\array_key_exists($default, $factories)) {
                     if (!\is_a($default, CollectionFactoryInterface::class, true)) {
                         $cfgPath[] = 'default';
-                        throw new \RuntimeException(\sprintf('Default collection factory `%s` not found.', $default));
+                        throw new RuntimeException(\sprintf('Default collection factory `%s` not found.', $default));
                     }
-                    $default = \is_string($default) ? $injector->make($default) : $default;
+                    $default = $injector->make($default);
```

Comment:
That change may be incorrect

Reply:
Here default should be string. If not, then throw error above in array_key_exists.

Reply:
It can be object too. And it will be correct.

Reply:
It cannot be objecte, because throw error here:

![image](https://user-images.githubusercontent.com/525501/214843036-5f79a1f6-330a-41c6-b588-3bbb27fa23f5.png)

See https://3v4l.org/Ff1B9

Reply:
Then integer

Reply:
it need to rewrite and add test

Reply:
I think we can skip it now